resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/exception/ApplicationAsyncExceptionHandler.java

```diff
@@ -33,7 +35,8 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(ApplicationAsyncExceptionHandler.class);
 
     @Override
-    public void handleUncaughtException(final Throwable throwable, final Method method, @Nullable final Object... objects) {
+    public void handleUncaughtException(final Throwable throwable, final Method method, @Nullable final Object[] objects) {
```

Comment:
Same a with the other example of this.  Three questions: @Nullable Object array in the method parameters but test that it is not null in the method call; Throwable and Method are assumed not null and not tested; Optional instead of Object test of null.

Reply:
Please see the response in the previous comment

Reply:
Updated

Reply:
Updated